### PR TITLE
MAINT, TST: default_rng few tests

### PR DESCRIPTION
**Pull request by tylerjereddy**
* I noticed two tests where we should now be able to safely use `default_rng`/`random.Generator` while reviewing some other code--I think we are well past the minimum NumPy version required to do this, and already do it in other tests

[skip cirrus] [skip circle]

### CODE REVIEW

**Review by h-vetinari — APPROVED** [review]

**Comment by ev-br** [comment]
LGTM, has a core dev approval and CI is passing module an unrelated hiccup. Merging, thanks Tyler.

**Pull request merged**
